charts/victoria-metrics-k8s-stack/templates/victoria-metrics-operator/alertmanager/vmalertmanager.yaml

```diff
@@ -6,7 +6,8 @@ kind: Secret
 metadata:
   name: {{ include "victoria-metrics-k8s-stack.fullname" . }}-alertmanager
   namespace: {{ .Release.Namespace }}
-  labels: 
+  labels:
+    alertmanager.config/checksum: {{ join "" .Values.alertmanager.config | sha256sum | trunc 63 | quote }}
```

Comment:
Operator must automatically sync configuration changes (it takes 3-5 mins). There is no need in adding checksum for it.